releng/org.eclipse.xtext.sdk.target/org.eclipse.xtext.sdk.target.target

```diff
@@ -156,11 +156,11 @@
 		<unit id="org.junit.platform.runner.source" version="1.3.1.v20181005-1442"/>
 		<unit id="org.opentest4j" version="1.1.1.v20181005-1442"/>
 		<unit id="org.opentest4j.source" version="1.1.1.v20181005-1442"/>
-		<unit id="org.objectweb.asm" version="6.2.1.v20180823-1831"/>
-		<unit id="org.objectweb.asm.source" version="6.2.1.v20180823-1831"/>
-		<unit id="org.objectweb.asm.tree" version="6.2.1.v20180823-1831"/>
-		<unit id="org.objectweb.asm.tree.source" version="6.2.1.v20180823-1831"/>
-		<repository location="http://download.eclipse.org/modeling/tmf/xtext/updates/orbit/2018-12"/>
+		<unit id="org.objectweb.asm" version="7.0.0.v20181030-2244"/>
+		<unit id="org.objectweb.asm.source" version="7.0.0.v20181030-2244"/>
+		<unit id="org.objectweb.asm.tree" version="7.0.0.v20181030-2244"/>
+		<unit id="org.objectweb.asm.tree.source" version="7.0.0.v20181030-2244"/>
+		<repository location="http://download.eclipse.org/tools/orbit/I-builds/I20181031064644/repository"/>
```

Comment:
Be careful with this line

Reply:
good catch was not intended. just for trying